types/jquery.notify/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for jquery.notify 1.5
+// Project: https://github.com/ehynds/jquery-notify (jQuery Notify UI Widget by Eric Hynds)
```

Comment:
Why not name this to jquery-notify to match the github name even though it's not on npm?